src/main/python/verification/controllers.py

```diff
@@ -24,24 +37,69 @@ def verify_id():
         URL: http://localhost:5000/verifyID
         ----------------------------------------------------------------------
         """
-    # image_of_id = request.files.get("id_img")
-    # names = request.form['names']
-    # surname = request.form['surname']
-    # id_number = request.form['id_number']
-    # nationality = request.form['nationality']
-    # country_of_birth = request.form['cob']
-    # status = request.form['status']
-    # gender = request.form['gender']
-    # date_of_birth = request.form['dob']
-    # face = request.files.get('face_img')
-    logger.debug(request)
+    data = {"success": False}
+    # Get id image as numpy array
+    # Check to see if an image was uploaded.
+    if request.files.get("id_img", None) is not None:
+        # Grab the uploaded image.
+        image_of_id = _grab_image(stream=request.files["id_img"])
+    # Otherwise, assume that a URL was passed in.
+    else:
+        # Grab the URL from the request.
+        url = request.args.get("url", None)
+        # If the URL is None, then return an error.
+        if url is None:
+            data["error"] = "No URL provided."
+            return jsonify(data)
+            # Load the image and convert.
+        image_of_id = _grab_image(url=url)
 
-    # do stuff to get result
+    # Get face image as numpy array
+    # Check to see if an image was uploaded.
+    if request.files.get("face_img", None) is not None:
+        # Grab the uploaded image.
+        face = _grab_image(stream=request.files["face_img"])
+    # Otherwise, assume that a URL was passed in.
+    else:
+        # Grab the URL from the request.
+        url = request.args.get("url", None)
+        # If the URL is None, then return an error.
+        if url is None:
+            data["error"] = "No URL provided."
+            return jsonify(data)
+            # Load the image and convert.
+        face = _grab_image(url=url)
+
+    entered_details = {
+        "names": request.form['names'], "surname": request.form['surname'], "identity_number":
```

Comment:
Change this dict so that each property is on its own line, for better readability